doc/source/admin/special_topics/interactive_environments.rst

```diff
@@ -9,7 +9,7 @@ How GIEs Work
 -------------
 
 A GIE is primarily composed of a Docker container, and the Galaxy visualization
-component. Galaxy vizualisation plugins are rendered using Mako templates and
+component. Galaxy visualisation plugins are rendered using Mako templates and
```

Comment:
s/visualisation/visualization   (I know, it's the en-US spelling, but that's what we've standardized on)